src/types/js-quantities/index.d.ts

```diff
@@ -0,0 +1,75 @@
+// Type definitions for JS-quantities 1.6

```

Comment:
Just add a comment that the version available on the DefinitelyTyped is not up to date, so we are adding a custom `.d.ts` for js-quantiles

Reply:
OK